_sass/code.scss

```diff
@@ -3,14 +3,16 @@
 
 // {% raw %}
 
-// This instruction applies to all queues not within 'pre', avoiding 'code' generated by the highlight.
+// This instruction applies to all queues not within 'pre' or 'figure', avoiding 'code' generated by the highlight.
 :not(pre) {
-  & > code {
-    padding: 0.2em 0.15em;
-    font-weight: 400;
-    background-color: $code-background-color;
-    border: $border $border-color;
-    border-radius: $border-radius;
+  &:not(figure) {
```

Comment:
Just got a bit more sleep, and am wondering - could this not just be

```css
:not(pre):not(figure) {

}
```

Reply:
stylelint doesn't like it

Reply:
Just looked into this; [selector-not-notation](https://stylelint.io/user-guide/rules/selector-not-notation/)'s `complex` feature is fine IMO, since the only major browser that [doesn't support Level 4 selectors is IE](https://caniuse.com/css-not-sel-list) (and I am perfectly fine not supporting IE anymore).

In this case, the correct code should be:

```css
:not(figure, pre) {
  /* ... */
}
```

Sorry to be so nitpicky here, just want to abide by our convention. Do you mind making this one last change? Then I'll give this the rubber stamp!